### PR TITLE
more pubs 2018/19: jfp, jar, tecs

### PR DESCRIPTION
**Pull request by ulysses4ever**

### CODE REVIEW

File: publications.rkt [hunk 1]
```diff
@@ -51,9 +51,21 @@
...
+    (publication "Verifying a Concurrent Garbage Collector with a Rely-Guarantee Methodology"
+                 "Yannick Zakowski, David Cachera, Delphine Demange, Gustavo Petri, David Pichardie, Suresh Jagannathan, and Jan Vitek"
+                 TECS
+                 2019
+                 "http://janvitek.org/pubs/tecs18.pdf")
+    (publication "How to Evaluate the Performance of Gradual Type Systems"
+                 "Ben Greenman, Asumu Takikawa, Max S. New, Daniel Felty, Robert Bruce Findler, Jan Vitek, and Matthias Felleisen"
+                 JFP
+                 2018
+                 "http://janvitek.org/pubs/jfp18.pdf")
```
Comment (bennn) [inline publications.rkt]: @janvitek  here's a link to the final version of the paper <http://www.ccs.neu.edu/home/types/publications/pe4gt/gtnffvf-jfp-2019.pdf>

(@ulysses4ever no need to change the link on the PRL publications page)

File: publications.rkt [hunk 2]
```diff
@@ -51,9 +51,21 @@
...
 
 @(define publications
    (list
+    (publication "Verifying a Concurrent Garbage Collector with a Rely-Guarantee Methodology"
+                 "Yannick Zakowski, David Cachera, Delphine Demange, Gustavo Petri, David Pichardie, Suresh Jagannathan, and Jan Vitek"
+                 TECS
+                 2019
+                 "http://janvitek.org/pubs/tecs18.pdf")
+    (publication "How to Evaluate the Performance of Gradual Type Systems"
+                 "Ben Greenman, Asumu Takikawa, Max S. New, Daniel Felty, Robert Bruce Findler, Jan Vitek, and Matthias Felleisen"
```
Comment (bennn) [inline publications.rkt]: `Felty` is misspelled --- please change to `Feltey`

Comment (ulysses4ever) [reply publications.rkt]: Many thanks for the review! Is it ok to fix the typo in `master` directly? I have run into this kind of issues before and have not found elegant way to augment a merged PR.

Comment (bennn) [reply publications.rkt]: Yeah absolutely. Add a new commit.

Don't try to change the history of commits already pushed to master.

Comment (ulysses4ever) [reply publications.rkt]: Done with b9f39f1bc2.

Comment (ulysses4ever) [reply publications.rkt]: And thanks again!

@janvitek This was copied from your homepage, so you might want to fix the typo in Feltey's last name (JFP'18) there too. Btw if you had the homepage repo hosted somewhere, I could even PR it.

File: publications.rkt [hunk 3]
```diff
@@ -51,9 +51,21 @@
...
 @(define ESECFSE "European Software Engineering Conference and Symposium on the Foundations of Software Engineering (ESEC/FSE)")
 @(define CSUR "ACM Computing Surveys (CSUR)")
+@(define JAR "Journal of Automated Reasoning (JAR)")
+@(define TECS "ACM Transactions on Embedded Computing Systems (TECS)")
 
 @(define publications
    (list
+    (publication "Verifying a Concurrent Garbage Collector with a Rely-Guarantee Methodology"
+                 "Yannick Zakowski, David Cachera, Delphine Demange, Gustavo Petri, David Pichardie, Suresh Jagannathan, and Jan Vitek"
+                 TECS
```
Comment (bennn) [inline publications.rkt]: `TECS` should be `JAR`

The `TECS` paper is "Can Android Run on Time?"

Comment (ulysses4ever) [reply publications.rkt]: Oh, my, I spoiled is heavily. Hope 7f41bf3 deals with this.

**Pull request merged**